samples/javascript/README.md

```diff
@@ -0,0 +1,37 @@
+# Export API Javascript Example
+
+In this JavaScript example, build durations are calculated and written out to the JavaScript console.
+
+## Setup
+
+To run this sample:
+
+1. Install [node.js].
+2. Run `npm install` within this directory.
+
+#### Create an Export API user for your Gradle Enterprise instance
+3. Create a Gradle Enterprise user with the `Export API` role as described in the [Export API Access Control] documentation.
+4. Set two environment variables locally: `EXPORT_API_USER` `EXPORT_API_PASSWORD` to match the newly created Export API user credentials.
+
+#### Running
```

Comment:
If `Setup` and `Authorisation` are used before, I would rename this step `Execution`.